doc/libpmemset/pmemset_drain.3.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEMSET_DRAIN, 3)
+collection: libpmemset
+header: PMDK
+date: pmemset API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2021, Intel Corporation)
+
+[comment]: <> (pmemset_drain.3 -- man page for libpmemset pmemset_drain function)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmemset_drain**() - drain flushes
+
+# SYNOPSIS #
+
+```c
+#include <libpmemset.h>
+
+int pmemset_drain(struct pmemset *set);
+```
+
+# DESCRIPTION #
+
+The **pmemset_drain**() funstion efficiently draining flushes in the range owned by *set*.
```

Comment:
I think we should provide at least some useful information as to what drain is instead of just redirecting to pmem2 man page.